types/string.go

```diff
@@ -82,3 +82,15 @@ func (s String) IsNull() bool {
 func (s String) IsUnknown() bool {
 	return s.Unknown
 }
+
+func (s String) String() string {
+	if s.Unknown {
+		return attr.UnknownString
+	}
+
+	if s.Null {
+		return attr.NullString
+	}
+
+	return s.Value
```

Comment:
It might be good to quote the value, e.g. `fmt.Sprintf("%q", s.Value)` as it may contain spaces, braces, commas, colons, etc that may interfere visually with other string outputs. Another odd case is an empty, but not null string causing the output to be like `[]` for a list with one empty string element, which I think is confusing.

Reply:
OK.